framework_tvb/tvb/adapters/analyzers/bct_adapters.py

```diff
@@ -169,25 +175,26 @@ def get_form_class(self):
         return BaseUnidirectedBCTForm
 
     @abstractmethod
-    def launch(self, connectivity, **kwargs):
+    def launch(self, view_model):
         pass
 
-
 class ModularityOCSM(BaseBCT):
     """
     """
     _ui_group = BCT_GROUP_MODULARITY
     _ui_connectivity_label = "Directed (weighted or binary) connection matrix:"
 
-    _ui_name = "Optimal Community Structure and Modularity"
+    _ui_name = "Optimal Community Structure and Modularity: Directed (weighted or binary) connection matrix"
```

Comment:
Wouldn't be better in english if we write something like:
'Compute optimal Community Structure and Modularity from a Directed (weighted or binary) connection matrix:`
I am wondering if the length is not too much, for the GUI to still display it nicely.